@LiveInLifeClient/addons/atcInterface/functions/fn_iterateMapUpdate.sqf

```diff
@@ -28,17 +28,16 @@ try
 		if (isNull _ui) throw false;
 		if !([player] call lilc_common_fnc_isAlive) throw false;
 
-		private _currentVehicles = (allMissionObjects "AllVehicles") select
-		{
-			private _vehicle = _x;
-			(isEngineOn _vehicle) &&
-			!(
-				((getPosATL _vehicle) select 2) < 45 &&
-				!(_vehicle getVariable ["lilc_transponder_status", false]) &&
-				(({ (_vehicle inArea _x) } count lilc_atcInterface_airports) <= 0)
-			) &&
-			(_vehicle isKindOf "Air")
-		};
+		  	{ 
```

Comment:
`_currentVehicles` is not defined.

Reply:
Try to use the new `select`-syntax instead of a simple `forEach` as you removed above.